### ISSUE TITLE
Sioyek crashes for documents with many pages

### ISSUE THREAD

**Issue by A3EKV** [ahrm/sioyek#108]
Hello,
When I open the following PDF document (**EDIT:** Link is removed because the text is protected by copyright) in sioyek and scrolling using the keyboard it will crashes when it reaches page 10 or 11 (this may differ). On the terminal I get the following error message:

```
sioyek: pdf_viewer/utils.cpp:727: bool are_stext_chars_far_enough_for_equation(fz_stext_char*, fz_stext_char*): Assertion `second_width >= 0' failed.
Aborted (core dumped)
```

**Sioyek version**: I am using the latest version (the latest commit: 094dd08  )
**System**: Ubuntu 20.04

Best regards

**Comment by ahrm** [comment]
Should be fixed in 1d5fa31c155615c9d4fce7a92b43038f0220010a.

**Comment by A3EKV** [comment]
Yes indeed, that fixes it.
